### PR TITLE
static header

### PR DESCRIPTION
**Pull request by ayusharma**
Static header as discussed here https://github.com/biojs/biojs.net/issues/19

### CODE REVIEW

**Comment by jayjayjpg** [comment]
Thanks for your PR @ayusharma - I'll check it out today

File: content/themes/biojs-theme/assets/css/main.css [hunk 1]
```diff
@@ -92,6 +92,13 @@ a[href] {
     width: 0%;
     height: 100%; }
 
+nav {
```
Comment (jayjayjpg) [inline content/themes/biojs-theme/assets/css/main.css]: @ayusharma For the changes being available to other developers as well, please do not change the main.css file which is recompiled anytime someone uses our project's [grunt](gruntjs.com) task to edit css. Instead, do your changes in the sass directory only, as described in our [README](https://github.com/biojs/biojs.net2#get-started-with-sass). If you are not familiar with SASS yet, the [SASS lang Documentation](http://sass-lang.com/) is a great place to get started.

**Comment by ayusharma** [comment]
@jessica-jordan ohh.. Sorry I forgot about saas.. Generated a new PR. Please take a look.

**Comment by daviddao** [comment]
@ayusharma Thanks! Can you squash your commits? Or just commit the changed sass file. Then I will merge.

**Comment by ayusharma** [comment]
@daviddao Please take a look.

**Comment by daviddao** [comment]
Thanks :)

**Pull request merged**